display_list/display_list_unittests.cc

```diff
@@ -1232,11 +1232,11 @@ TEST(DisplayList, FlutterSvgIssue661BoundsWereEmpty) {
   // improved and maintained, then we can eliminate this test and
   // just rely on the "rounded out" bounds test that follows.
   EXPECT_EQ(display_list->bounds(),
-            SkRect::MakeLTRB(0, 0.00189208984375, 99.9839630127, 100));
+            SkRect::MakeLTRB(0, 0.00190162658691406, 99.9839553833008, 100));
```

Comment:
A note to self: Figure out why these bounds changed subtly. Nothing about the new code should have caused these bounds to change AFAICT, but the test doesn't pass without this edit.

Reply:
It looks like this change is causing failures in the presubmit checks. I'm double checking, but it looks like my machine will fail this test without this change to the expected rect in this test, but the test machines are perfectly happy with the old values. This may be an M1 vs Intel discrepancy that we'll have to get to the bottom of or work around it.

Reply:
Fixed in https://github.com/flutter/engine/pull/38435